### ISSUE TITLE
undefined symbol: _ZTIN3c1021AutogradMetaInterfaceE".that's why?

### ISSUE THREAD

**Issue by YanDingXin** [princeton-vl/CornerNet-Lite#69]
Hello, author.  When I tried to run 'demo.py', the following error occurred.  "ImportError: /home/lotvs/.local/lib/python3.7/site-packages/cpools-0.0.0-py3.7-linux-x86_64.egg/top_pool.cpython-37m-x86_64-linux-gnu.so: undefined symbol: _ZTIN3c1021AutogradMetaInterfaceE"。  I have already completed some preparatory work according to your instructions before running demo.  I hope you can reply to me.  Thank you again.

**Comment by SeeeeShiwei** [comment]
Solve it？

**Comment by YanDingXin** [comment]
> Solve it？

Yes, I have solved this problem.You can go to this [website](https://github.com/YanDingXin/CornerNet-Lite-core) and download the <core> file and replace your original one.Then do python setup.py...And make will run successfully.(it works on python3.7, pytorch1.0) I guess it was the author who changed C's API interface that caused the error.

**Comment by njustymk** [comment]
it didn't work
ImportError: /root/.local/lib/python3.6/site-packages/cpools-0.0.0-py3.6-linux-x86_64.egg/top_pool.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZN3c105ErrorC1ENS_14SourceLocationERKSs

**Comment by gupengju** [comment]
> it didn't work
> ImportError: /root/.local/lib/python3.6/site-packages/cpools-0.0.0-py3.6-linux-x86_64.egg/top_pool.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZN3c105ErrorC1ENS_14SourceLocationERKSs

maybe you need to install the right version of cudnn

**Comment by qiangruoyu** [comment]
> it didn't work
> ImportError: /root/.local/lib/python3.6/site-packages/cpools-0.0.0-py3.6-linux-x86_64.egg/top_pool.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZN3c105ErrorC1ENS_14SourceLocationERKSs

Solve it？

**Comment by caikw0602** [comment]
Yes, the same error. Please help.

**Comment by YanDingXin** [comment]
> Yes, the same error. Please help.

I've solved the problem. You can try to do as I said：You can go to this[https://github.com/YanDingXin/CornerNet-Lite-core](url) website and download the file and replace your original one.Then do python setup.py...And make will run successfully.(it works on python3.7, pytorch1.0) I guess it was the author who changed C's API interface that caused the error.

**Comment by GKCsdfsGKC** [comment]
> > Yes, the same error. Please help.
> 
> I've solved the problem. You can try to do as I said：You can go to this[https://github.com/YanDingXin/CornerNet-Lite-core](url) website and download the file and replace your original one.Then do python setup.py...And make will run successfully.(it works on python3.7, pytorch1.0) I guess it was the author who changed C's API interface that caused the error.

solved, thank you!

**Comment by Maulik1528** [comment]
> > Yes, the same error. Please help.
> 
> I've solved the problem. You can try to do as I said：You can go to this[https://github.com/YanDingXin/CornerNet-Lite-core](url) website and download the file and replace your original one.Then do python setup.py...And make will run successfully.(it works on python3.7, pytorch1.0) I guess it was the author who changed C's API interface that caused the error.

Given link is not working.
